core/src/main/java/org/operationcrypto/hivej/communication/ConnectionManager.java

```diff
@@ -32,9 +33,11 @@
  */
 public class ConnectionManager {
     private static final Logger LOGGER = LoggerFactory.getLogger(ConnectionManager.class);
+    /** The default endpoint URI */
+    private static final String DEFAULT_HIVE_API_URI = "https://api.hive.blog/";
```

Comment:
we also do not destinguish between static and member variables in terms of naming conventions? (e.g. prefix 's' for static variables?)